### ISSUE TITLE
Add installation instructions on README.md

### ISSUE THREAD

**Issue by desyncr** [desyncr/geometry-dir-info-prompt#4]

**Comment by mustaqimM** [comment]
Hi there @desyncr How do I use the plugin? I cloned it and put it in the `plugins` folder and had it set:
```bash
▲ ~ echo ${GEOMETRY_PROMPT_PLUGINS:?}
virtualenv docker_machine exec_time git hg geometry-dir-info-prompt
```
but it still doesn't load.

**Comment by desyncr** [comment]
Well, it seems I won't load at all. I'll take a look to see what's going on.

**Comment by desyncr** [comment]
It seems it's an issue with latest geometry. As a workaround use geometry `v1.0.0`.

**Comment by desyncr** [comment]
@mustaqimM I'll open up an issue on geometry to enhance documentation on installing custom plugins.

**Comment by desyncr** [comment]
Should be fixed with #7
